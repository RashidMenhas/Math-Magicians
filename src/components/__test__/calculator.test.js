import { render } from '@testing-library/react';
import Calculator from '../calculator';

it('test for Calculator compoenent', () => {
  const tree = render(<Calculator />);
  expect(tree).toMatchSnapshot();
});
