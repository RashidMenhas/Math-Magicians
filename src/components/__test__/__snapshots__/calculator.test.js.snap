// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for Calculator compoenent 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="calculator"
      >
        <h3>
          Lets do some math!
        </h3>
        <div
          class="container"
        >
          <div
            class="result-area"
          >
            <p
              class="result"
            >
              0
            </p>
          </div>
          <div
            class="All-btns"
          >
            <button
              class="AC"
              type="button"
            >
              AC
            </button>
            <button
              class="PM"
              type="button"
            >
              +/-
            </button>
            <button
              class="modules"
              type="button"
            >
              %
            </button>
            <button
              class="div gold"
              type="button"
            >
              รท
            </button>
            <button
              class="seven"
              type="button"
            >
              7
            </button>
            <button
              class="eigth"
              type="button"
            >
              8
            </button>
            <button
              class="nine"
              type="button"
            >
              9
            </button>
            <button
              class="x gold"
              type="button"
            >
              x
            </button>
            <button
              class="four"
              type="button"
            >
              4
            </button>
            <button
              class="five"
              type="button"
            >
              5
            </button>
            <button
              class="six"
              type="button"
            >
              6
            </button>
            <button
              class="minus gold"
              type="button"
            >
              -
            </button>
            <button
              class="one"
              type="button"
            >
              1
            </button>
            <button
              class="two"
              type="button"
            >
              2
            </button>
            <button
              class="three"
              type="button"
            >
              3
            </button>
            <button
              class="plus gold"
              type="button"
            >
              +
            </button>
            <button
              class="zero"
              type="button"
            >
              0
            </button>
            <button
              class="dot"
              type="button"
            >
              .
            </button>
            <button
              class="equal gold"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="calculator"
    >
      <h3>
        Lets do some math!
      </h3>
      <div
        class="container"
      >
        <div
          class="result-area"
        >
          <p
            class="result"
          >
            0
          </p>
        </div>
        <div
          class="All-btns"
        >
          <button
            class="AC"
            type="button"
          >
            AC
          </button>
          <button
            class="PM"
            type="button"
          >
            +/-
          </button>
          <button
            class="modules"
            type="button"
          >
            %
          </button>
          <button
            class="div gold"
            type="button"
          >
            รท
          </button>
          <button
            class="seven"
            type="button"
          >
            7
          </button>
          <button
            class="eigth"
            type="button"
          >
            8
          </button>
          <button
            class="nine"
            type="button"
          >
            9
          </button>
          <button
            class="x gold"
            type="button"
          >
            x
          </button>
          <button
            class="four"
            type="button"
          >
            4
          </button>
          <button
            class="five"
            type="button"
          >
            5
          </button>
          <button
            class="six"
            type="button"
          >
            6
          </button>
          <button
            class="minus gold"
            type="button"
          >
            -
          </button>
          <button
            class="one"
            type="button"
          >
            1
          </button>
          <button
            class="two"
            type="button"
          >
            2
          </button>
          <button
            class="three"
            type="button"
          >
            3
          </button>
          <button
            class="plus gold"
            type="button"
          >
            +
          </button>
          <button
            class="zero"
            type="button"
          >
            0
          </button>
          <button
            class="dot"
            type="button"
          >
            .
          </button>
          <button
            class="equal gold"
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
